### DISCUSSION TITLE
File count per store item too high

### DISCUSSION THREAD

**Discussion by chriscross1211** [Courseplay/Courseplay_FS22#2636] [Q&A]
What could be the cause of the "File count per store item too high" error message when attempting to upload Courseplay to our server?

**Comment by Tensuko** [comment]
Thats no error, its a warning and can be ignored. That got no impact.

**Reply by XxBeth2210xX** [reply]
this is showing me too but when i just activate it and start the server its not there.

**Reply by Tensuko** [reply]
Logfile?

**Reply by ciesarik3** [reply]
I upload and activate mod on 4net players server and it loads 1/3 of mode i see in menu 2 tabs missing on side menu. I paste a log file from PC 
[log.txt](https://github.com/user-attachments/files/16341317/log.txt)
and log from server 
[server log.txt](https://github.com/user-attachments/files/16341336/server.log.txt)
![Screenshot_24](https://github.com/user-attachments/assets/ef1397a3-e204-4672-8405-52682b7d4942)

THX for reply

**Reply by Tensuko** [reply]
Nothing is missing there.
Do you even sit in a vehicle?

**Reply by ciesarik3** [reply]
my bad thx
